src/components/FocusVariantHeaderControls.tsx

```diff
@@ -101,8 +104,7 @@ export const FocusVariantHeaderControls = React.memo(({ selector }: Props): JSX.
 
   const listLink2: string | undefined =
     listLink &&
-    listLink.replace('contributors', 'gisaid-epi-isl').replace('&downloadAsFile=true&dataFormat=csv', '') +
-      '&orderBy=random';
```

Comment:
@chaoran-chen LAPIS 2 doesn't support this. Is it ok to remove it?

Reply:
As discussed, we have now a new ticket, to introduce orderBy=random, because we need it for the usher integration:
#927